src/main/java/ch/uzh/ifi/hase/soprafs22/service/UserService.java

```diff
@@ -65,6 +65,7 @@ public User createUser(User newUser) {
     newUser.setMinAge(findMinAgeDefault(newUser.getBirthday()));
     newUser.setMaxAge(findMaxAgeDefault(newUser.getBirthday()));
     Set<Gender> genderPreferences = new TreeSet<>();
+    newUser.setAge(getAge(newUser.getBirthday()));
```

Comment:
Where do you update the age? If you REALLY want to add an age column (again, I think not needed), then you must add a function that updates it (When to call that function?, who calls it?,...)

Reply:
I added it to the user whenever the user logs in. This partially fixes it. I still think it's more sensible, as this also makes the query simpler (otherwise the other user also doesn't have an age, which you would then have to pass to the query)

Reply:
Logically, if I log in today, have birthday tomorrow, and jump out of your agePreference, you would still get a game from me eventhough I will not have logged in tomorrow.

Reply:
Yes, but the same argument would have to be made for age preferences, as they get set as +- 3 years of birthday and don't get updated. So i think in this very inconsequential edgecase it shouldn't really be a problem

Reply:
Well, no. Your age updates automatically, wheras your preferrenced age does not. Plus it is only an edge case because the game expires after one day. If we would change it to one month or so, it would matter and debugging the code would be harder.

Reply:
-Irrelevant with new implementation (age no longer exists)